scripts/change_alias.py

```diff
@@ -1,7 +1,38 @@
+from constants import INDICES, STAGING_NODE1
 from elasticsearch import Elasticsearch
-import argparse
+import click
+import logging
 
-from utils import *
+from utils import remove_underscore_from_end_prefix
+
+logger = logging.getLogger(__name__)
```

Comment:
logger instances should be created inside script entry point (if __name__=="__main__"), as we don't need to create these instances every time we import this script from another script, we only need to do this when we actually run this script.

Reply:
big action may be needed to improve the utils function:
for this script, it is better to display messages to the screen rather than the file

Reply:
Then why don't you use just print?

Reply:
logging more flexible, e.g. set logger level to warning, these message will be hidden.
current logging  setting also provides line number, useful to debug and timestamp

Reply:
agree to add parameter to create_logging_instance function in utils to switch between log file and screen output

Reply:
done in 9ba00dd